app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,72 @@
+class TasksController < ApplicationController
+  def index
+    @tasks = Task.all
+  end
+
+  def new
+    @task = Task.new
+  end
+
+  def create
+    task = Task.new(task_params)
+
+    if task.save
+      redirect_to "/tasks"
```

Comment:
This works for now, but you'll want to be using the `_path` routes going forward